python/pyfunc-server/pyfuncserver/protocol/rest/server_sanic.py

```diff
@@ -0,0 +1,93 @@
+import json
+import logging
+
+from merlin.pyfunc import PyFuncOutput
+from orjson import dumps, loads
+from prometheus_client import CollectorRegistry
+from prometheus_client.exposition import choose_encoder
+from sanic import Sanic
+from sanic import json as sanic_json
+from sanic import raw, text
+from sanic.request import Request
+
+from pyfuncserver.config import Config
+from pyfuncserver.model.model import PyFuncModel
+
+
+class HTTPServer:
+    def __init__(
+        self, config: Config, metrics_registry: CollectorRegistry, model: PyFuncModel
+    ):
+
+        self.config = config
+        self.workers = config.workers
+        self.model_manifest = config.model_manifest
+        self.http_port = config.http_port
+        self.metrics_registry = metrics_registry
+        self.registered_models: dict = {}
+        self.register_model(model)
+
+        Sanic.start_method = "fork"
```

Comment:
in Tornado, `self._http_server.start(self.workers)` will use `fork` mechanism

![image](https://github.com/caraml-dev/merlin/assets/30390872/dbfa69be-781a-44bb-b9f2-090ef3ec418b)